buildSrc/src/main/java/Gecko.kt

```diff
@@ -3,7 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 private object GeckoVersions {
-    const val nightly_version = "67.0.20190219094627"
+    const val nightly_version = "67.0.20190304093752"
```

Comment:
eek do we need to do this with every version bump?

Reply:
@liuche nope! just trying to stay up to date for now